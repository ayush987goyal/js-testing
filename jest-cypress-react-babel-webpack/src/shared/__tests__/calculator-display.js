import React from 'react'
import {render} from '@testing-library/react'
import {createSerializer} from 'jest-emotion'
import * as emotion from '@emotion/core'
import CalculatorDisplay from '../calculator-display'

expect.addSnapshotSerializer(createSerializer(emotion))

test('mounts', () => {
  const {container} = render(<CalculatorDisplay value="0" />)
  expect(container.firstChild).toMatchSnapshot()
})
