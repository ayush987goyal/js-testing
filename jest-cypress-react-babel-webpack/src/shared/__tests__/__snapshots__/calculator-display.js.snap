// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mounts 1`] = `
.emotion-0 {
  position: relative;
  color: white;
  background: #1c191c;
  line-height: 130px;
  font-size: 6em;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

<div
  class="emotion-0"
>
  <div
    class="autoScalingText"
    data-testid="total"
    style="transform: scale(1,1);"
  >
    0
  </div>
</div>
`;
